### PR TITLE
[DEV-185] SEPA Instant

### PR DESCRIPTION
**Pull request by davidleomay**
Only available in german and english ... all other languages will be added later.

### CODE REVIEW

File: app/screens/home/RouteList.tsx [hunk 1]
```diff
@@ -172,12 +183,12 @@ const RouteList = ({
...
   };
   const deleteCryptoRoute = (route: CryptoRoute) => {
-    setIsCryptoLoading((obj) => updateObject(obj, { [route.id]: true }))
+    setIsCryptoLoading((obj) => updateObject(obj, { [route.id]: true }));
     return putCryptoRoute(updateObject(route, { active: false }))
       .then(() => (route.active = false))
       .catch(() => NotificationService.error(t("feedback.delete_failed")))
       .finally(() => setIsCryptoLoading((obj) => updateObject(obj, { [route.id]: false })));
-  }
+  };
```
Comment (Krysh90) [inline app/screens/home/RouteList.tsx:191]: Maybe it is just me and coming from other languages, but using `;` feels weird. Most languages neglected this usage as it is not needed. I personally heavy prefer to not use it either

File: app/screens/home/RouteList.tsx [hunk 2]
```diff
@@ -270,32 +281,32 @@ const RouteList = ({
...
+            onPress: () => ClipboardService.copy(route.bankUsage),
+          },
+          {
+            condition: true,
+            label: "model.route.fee",
+            value: `${route.fee}%` + (route.refBonus ? ` (${route.refBonus}% ${t("model.route.ref_bonus")})` : ""),
+          },
+          { condition: true, label: "model.route.volume", value: `${formatAmount(route.volume)} €` },
+          { condition: true, label: "model.route.annual_volume", value: `${formatAmount(route.annualVolume)} €` },
+        ];
```
Comment (Krysh90) [inline app/screens/home/RouteList.tsx:309]: I guess I need to check why my formatting on save is not working with Prettier formatter

File: app/screens/home/RouteList.tsx [hunk 3]
```diff
@@ -721,29 +729,66 @@ const RouteList = ({
...
+
+          <View style={device.SM && [AppStyles.containerHorizontal, styles.sepaContainer]}>
+            <View style={[styles.sepaItem, device.SM && { flexShrink: 1 }]}>
+              <H4 text={t("model.sepa.instant.title")} style={{ color: Colors.Primary }} />
+              <CopyLine text={`${t("model.route.iban")}: ${instantIban}`} copyText={instantIban} />
+              <CopyLine text={`BIC: ${instantBic}`} copyText={instantBic} />
+              <SpacerV />
+              {[...Array(7).keys()].map((_, i) => (
+                <View key={i} style={[AppStyles.containerHorizontal, { alignItems: "flex-start" }]}>
+                  <View style={i > 3 && { marginLeft: 20 }}>
```
Comment (Krysh90) [inline app/screens/home/RouteList.tsx:743]: Would it make sense to move those two "random" values:
- length of 7
- other style on index greater than 3

to some more describing variables?

**Review by Krysh90 — APPROVED** [review]

File: app/screens/home/RouteList.tsx (hunk 2 above)
Comment (davidleomay) [reply app/screens/home/RouteList.tsx:309]: Yes 🤔

File: app/screens/home/RouteList.tsx (hunk 1 above)
Comment (davidleomay) [reply app/screens/home/RouteList.tsx:191]: That's default Prettier setting. Feel free to define custom rules and impose them on us (as long as it is auto formatted) ... I'm open for everything 😁

Comment (Krysh90) [reply app/screens/home/RouteList.tsx:186]: hahhaha oh shit, you are right. i mean if i haven't to write them. I don't really care. Lets see what the future brings in

File: app/screens/home/RouteList.tsx (hunk 2 above)
Comment (Krysh90) [reply app/screens/home/RouteList.tsx:304]: done and somehow it was deactivated

**Review by Krysh90 — APPROVED** [review]

**Pull request merged**